[feat-5] 모든 Error Function Customize 지원하기

akshxl09 opened the pull request:
모든 error에 대해서 customizing이 가능하도록 변경

JWT_CONFIG에 추가하는 방식으로 error customizing 지원

akshxl09 committed:
feat: error customizing

모든 error에 대해서 customizing 가능하도록 수정

iml1111 commented on django_jwt_extended/config.py:
```diff
@@ -56,4 +62,50 @@ def validate_refresh_token_expires(config: dict):
+    @staticmethod
+    def customize_jwt_not_found(config: dict):
+        jwt_not_found = config.get('JWT_NOT_FOUND_MSG', 'JWT token not found')
+        return {'msg': jwt_not_found}
```
메시지 하나(string)가 아닌 dict를 통째로 customize 가능하게 해주셈.
또한 해당 인자를 입력받았을 때 해당인자가 json serialize가 가능한지도 validate해주셈.
물론 밑에 모든 customize 메소드에 동일하게 적용해주셈.

iml1111 replied:
추가적으로, 모든 customize 메소드가 로직 자체는 굉장히 유사한 걸 알 수 있음.
아예 인자를 빼서 for문을 돌리면서 함수를 하나로 만들어 버리는 것도 괜찮을 것 같음.

akshxl09 committed:
update: string -> dict, customizing 로직 통합

string이 아닌 dict를 customizing 하도록 변경
json serializable한지 검증
전체 error customizing  메소드 통합

akshxl09 committed:
update: test 추가

print -> assertEqual로 변경
